codespell_lib/data/dictionary_rare.txt

```diff
@@ -120,6 +120,7 @@ habitants->inhabitants
 hart->heart, harm,
 heighth->height, eight, eighth,
 heighths->heights, eights, eighths,
+hep->heap, help,
```

Comment:
I wasn't aware that [hep](https://www.oed.com/search/dictionary/?q=hep) is a word in English.